src/v2/ion/i18nTransformer.js

```diff
@@ -238,6 +239,15 @@ const getMessage = (message) => {
   return message.message;
 };
 
+/**
+ * - iff `message.i18n.key` exists return the key.
+ *
+ * @param {Message} message
+ */
+const getMessageKey = (message) => {
+  return message.i18n?.key || '';
```

Comment:
Are we sure `message` will always be not null

Reply:
I think so.. getMessage also assumes it. But i think i will change this. Nice catch! 👍